geode-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -80,7 +80,7 @@ public RegionProvider(InternalCache cache) {
   protected static void validateBucketCount(int buckets) {
     if (buckets > REDIS_SLOTS) {
       throw new ManagementException(String.format(
-          "Could not start Redis Server - System property '%s' must be <= %d",
+          "Could not start Server Compatible with Redis - System property '%s' must be <= %d",
```

Comment:
Could not start server compatible with Redis - System property '%s' must be <= %d